libsandbox/wrappers.c

```diff
@@ -10,6 +10,10 @@
  *  as some of the InstallWatch code was used.
  */
 
+#undef _FILE_OFFSET_BITS
+#undef _TIME_BITS
+#define _LARGEFILE64_SOURCE
```

Comment:
Add in LARGEFILE_SOURCE too for good measure, perhaps?

Reply:
I checked for any usage of ftello or fseeko and could not find any. I'm pretty sure _LARGEFILE_SOURCE is not needed.

Reply:
OK, sure.